packages/hoppscotch-cli/src/utils/collections.ts

```diff
@@ -3,16 +3,30 @@ import * as A from "fp-ts/Array";
 import { pipe } from "fp-ts/function";
 import { bold } from "chalk";
 import { log } from "console";
+import { round } from "lodash";
```

Comment:
```suggestion
import round from "lodash/round";
```

Reply:
Shall we do this ?
As, in future we may use more function from `lodash`, so importing using destructuring syntax shouldn't be a issue ?

Reply:
This is more for keeping the code consistent between all the repos on the monorepo.
On `hoppscotch-app` for example, we use this style of importing to avoid any lodash functions we don't use being added into the bundle. This is not exactly a worry for the CLI, but it would be cool if it is consistent.